README.md

```diff
@@ -117,6 +117,7 @@ REDIS\_DB | 0 | Which database on the  redis server should be used
 REDIS\_PASSWORD | *empty* | Password to authenticate at the redis server
 CLEANUP\_INTERVAL | 900 | Interval (in seconds) of searching for abandoned rooms
 CLEANUP\_ROOM\_EXPIRE\_TIME | 3600 | Time (in seconds) until an empty room gets abandoned
+CLEANUP_ROOM_LIFE_TIME | 86400 | Time (in seconds) until an active room gets abandoned
```

Comment:
I think something like *MAX_ROOM_LIFE_TIME* would be much clearer.

Reply:
Okay